wolox-books/src/app/models/user.ts

```diff
@@ -0,0 +1,24 @@
+export class User {
```

Comment:
Actually, we use models just to document and set a type for our variables. You can use either `type` or `interface`, classes are not really used since the only classes we want (and need) are the components, services, etc. In this case:

```ts
interface User {
  name: string;
  lastName: string;
  email: string;
  password: string;
}
```